parents/integrationtest/pom.xml

```diff
@@ -140,33 +140,56 @@
                     </executions>
                 </plugin>
                 <plugin>
-                    <groupId>com.github.alexcojocaru</groupId>
-                    <artifactId>elasticsearch-maven-plugin</artifactId>
-                    <version>${version.com.github.alexcojocaru.elasticsearch.plugin}</version>
+                    <groupId>io.fabric8</groupId>
+                    <artifactId>docker-maven-plugin</artifactId>
+                    <version>${version.docker.maven.plugin}</version>
                     <configuration>
                         <skip>${test.elasticsearch.run.skip}</skip>
-                        <flavour>${test.elasticsearch.run.flavour}</flavour>
-                        <clusterName>hsearchEsTestCluster</clusterName>
-                        <httpPort>9200</httpPort>
-                        <version>${test.elasticsearch.connection.version}</version>
-                        <timeout>90</timeout><!-- Make it work on our slow CI -->
-                        <!-- auto-create-index is necessary for non-OSS Elasticsearch -->
-                        <autoCreateIndex>true</autoCreateIndex>
-                        <environmentVariables>
-                            <JAVA_HOME>${test.elasticsearch.run.java_home}</JAVA_HOME>
```

Comment:
So you're not using `test.elasticsearch.run.java_home` anymore. That makes sense, of course. But please finish removing it from the `README` and `Jenkinsfile`? Everything related to `elasticsearchJdkTool` in the Jenkinsfile, in particular, is no longer relevant.